src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/changes/benificialowner/psc/IndividualBeneficialOwnerPsc.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.benificialowner.psc;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.LocalDate;
+import java.util.Objects;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.PersonName;
+
+public class IndividualBeneficialOwnerPsc extends Psc {
+  @JsonProperty("personName")
+  private PersonName personName;
+
+  @JsonProperty("nationality")
```

Comment:
Is this property needed, I can't see it in the mapping documentation

Reply:
So it looks like this was changed on Friday where both nationality and nationalityOther are going to be in the same variable 

Again this is one that may be a significant setback as it changed the logic and tests that have already been written

[Before](https://companieshouse.atlassian.net/wiki/pages/viewpage.action?pageId=4144005155&pageVersion=66)
[After](https://companieshouse.atlassian.net/wiki/pages/viewpage.action?pageId=4144005155&pageVersion=66)